third_party/spdk_defs/nvme_defs.h

```diff
@@ -243,7 +243,36 @@ struct SglDescriptor {
 } ABSL_ATTRIBUTE_PACKED;
 static_assert(sizeof(SglDescriptor) == 16);
 
-} // namespace nvme_defs
+// NVMe Base Specification Figure 7
+// https://nvmexpress.org/wp-content/uploads/NVM-Express-1_4-2019.06.10-Ratified.pdf
+struct GenericQueueEntryCmd {
+  // dword 0
```

Comment:
I looked at the reference and it seems like dwords should be listed in big endian?

Reply:
Sorry, I messed up my documentation versions causing some confusion. Table 105 at https://nvmexpress.org/wp-content/uploads/NVM-Express-1_4a-2020.03.09-Ratified.pdf should clear this up